hasher-matcher-actioner/README.md

```diff
@@ -49,13 +49,20 @@ HMA requires a configuration file passed as an environment variable, `OMM_CONFIG
 
 #### Running the Application in Development
 
-To run HMA in a development environment using Docker, use the following command:
+To run HMA in a development environment using Docker, we first need to run the database migrations:
 
 ```bash
-$ docker run -e OMM_CONFIG='/build/reference_omm_configs/development_omm_config.py' -p 5000:5000 ghcr.io/facebook/threatexchange/hma flask --app OpenMediaMatch.app run --host=0.0.0.0
+$ docker run -e OMM_CONFIG='/reference_configs/development_omm_config.py' ghcr.io/facebook/threatexchange/hma /app/scripts/db-migrate.sh
 ```
 
-This command sets the necessary environment variable and exposes the app on port 5000 of your host machine, making the API accessible locally.
+
+Then we can start the server with:
+
+```bash
+$ docker run -e OMM_CONFIG='/reference_configs/development_omm_config.py' -p 5100:5100 ghcr.io/facebook/threatexchange/hma
```

Comment:
This wouldn't be quite right for development, since we're relying on a built image, we may be better suggesting devcontainers here?

Reply:
You can change this command by mounting the volume. 
```
--mount type=bind
```

This will make it such that changes on local will also change the container. I would also recommend instead using Docker compose as it will have both the app container and the postgresql container.

Reply:
We've already covered "developing HMA" in the `CONTRIBUTING.md` file, so I don't think the `README.md` needs any further changes here.